website/versioned_docs/version-3.4.0/concepts-shops.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Shops
+id: version-3.4.0-concepts-shops
+original_id: concepts-shops
+---
+
+The Reaction Commerce API is a **multi-shop system**, which means that more than one shop can exist within the same installation. Most API calls require you to specify a shop or shops. But what exactly is a shop?
+
+While you might think you intuitively know what a “shop” is, things can become fuzzy when there are complex scenarios like multiple frontend apps, or shops that sell items from other shops. In fact, it is really up to you to decide what you want a “shop” to be in your system. But there are also some rules you have to know to make this decision wisely.
+
+In general, we recommend that you create **shops** to be unique groupings of items and/or payment settings. In doing so, be aware of the following:
```

Comment:
With the minimal multi-shop support in the official release, I'm not confident we should be making recommendations for how to configure multi-shop setups in the docs. Perhaps docs and best-practices for a multi-shop configuration would be better contained in the docs for a community led plugin (e.g. the plugin that you/outgrow are working on) or in a community blog post?

Reply:
This is actually what the docs currently read on this page. I believe @machikoyasuda wrote this. Do you want to remove that paragraph along with the list that follows altogether? I can definitely do that, and I think it would be less confusing indeed.

Reply:
I didn't realize that this is what the docs currently read, but yes, let's try to make it less confusing, even if that means removing some existing work.